hw/ip/aon_timer/data/aon_timer.hjson

```diff
@@ -95,6 +95,43 @@
       width:   "1"
     }
   ],
+  features: [
+    { name: "AON_TIMER.WAKEUP",
+      desc: '''
+            The AON_TIMER tick period can be configured.
```

Comment:
This would go in the config sub.feature, and it is confusing to say the tick period. I think it is clearer to say something like  `Configures wakeup to happen after a number of AON clock cycles.`